apps/coordinator/vite.config.ts

```diff
@@ -41,6 +41,16 @@ export default defineConfig({
         (process.env.__VERCEL_GIT_COMMIT_SHA__ || "").slice(0, 7),
     ),
     __APP_VERSION__: JSON.stringify(process.env.npm_package_version),
+    "process.env": {
+      TREZOR_DEV: process.env.TREZOR_DEV,
+      __GIT_SHA__: JSON.stringify(
+        process.env.__GIT_SHA__ ||
+          (process.env.__VERCEL_GIT_COMMIT_SHA__ || "").slice(0, 7),
+      ),
+      __APP_VERSION__: JSON.stringify(process.env.npm_package_version),
+      TREZOR_CONNECT_URL: process.env.TREZOR_CONNECT_URL,
+      TREZOR_BLOCKBOOK_URL: process.env.TREZOR_BLOCKBOOK_URL,
```

Comment:
Yeah, which makes sense. It's building a final bundle and needs to fill in these values. But this wasn't the primary cause of the issue. The core of the problem was because the process.env was being used by a dependency (@caravan/wallets) which was getting built itself before being used in the vite app. that build process was stripping out the node globals.